homeassistant/components/fan/services.yaml

```diff
@@ -211,3 +211,17 @@ wemo_reset_filter_life:
     entity_id:
       description: Names of the WeMo humidifier entities (1 or more entity_ids are required).
       example: 'fan.wemo_humidifier'
+
+airscape_speed_up:
```

Comment:
Move these new entries to the `services.yaml` file in the airscape integration package.